packages/odyssey-react/src/components/Checkbox/Checkbox.module.scss

```diff
@@ -46,58 +46,61 @@
   &:indeterminate {
     + .label {
       .indicator {
-        color: var(--ColorCheck);
+        color: var(--IndicatorColor);
       }
     }
   }
 
   &:focus {
     + .label {
       .box {
-        @include outline(var(--ColorFocusOutline), var(--FocusOutlineWidth));
+        @include outline(
+          var(--BoxFocusOutlineColor),
+          var(--BoxFocusOutlineWidth)
+        );
 
-        border-color: var(--ColorBorderFocus);
+        border-color: var(--BoxFocusBorderColor);
       }
     }
   }
 
   &:disabled {
     + .label,
     + .label:hover {
-      color: var(--ColorTextDisabled);
+      color: var(--DisabledTextColor);
```

Comment:
`LabelDisabledTextColor` ? That is the approach I took in my Radio PR, gave Label it's own "part" name